catalog/app/containers/Bucket/PackageCreateDialog.js

```diff
@@ -24,42 +20,20 @@ import * as s3paths from 'utils/s3paths'
 import { readableBytes } from 'utils/string'
 import * as validators from 'utils/validators'
 
-import SelectWorkflow from './SelectWorkflow'
+import * as PD from './PackageDialog'
 import * as requests from './requests'
 
-const MAX_SIZE = 1000 * 1000 * 1000 // 1GB
-const ES_LAG = 3 * 1000
-
-const Errors = {
+const ERRORS = {
   FILES_UPLOAD: { [FORM_ERROR]: 'Error uploading files' },
   PACKAGE_CREATION: { [FORM_ERROR]: 'Error creating manifest' },
   PACKAGE_VALIDATION: { [FORM_ERROR]: 'Error validating package' },
 }
 
 const getNormalizedPath = (f) => (f.path.startsWith('/') ? f.path.substring(1) : f.path)
 
-function Field({ input, meta, errors, label, ...rest }) {
-  const error = meta.submitFailed && meta.error
-  const validating = meta.submitFailed && meta.validating
-  const props = {
-    error: !!error,
-    label: (
-      <>
-        {error ? errors[error] || error : label}
-        {validating && <M.CircularProgress size={13} style={{ marginLeft: 8 }} />}
-      </>
-    ),
-    disabled: meta.submitting || meta.submitSucceeded,
-    InputLabelProps: { shrink: true },
-    ...input,
-    ...rest,
-  }
-  return <M.TextField {...props} />
-}
-
 const useFilesInputStyles = M.makeStyles((t) => ({
   root: {
-    marginTop: t.spacing(2),
+    marginTop: t.spacing(3),
```

Comment:
Now all form sections have different margin :)

Header
32px
Name
24px
Commit message
32px
Files
24px
Metadata
16px
Workflow
40px
Buttons

Reply:
actually, i tried to unify the spacings by this change. the problem is almost all the inputs have their own margins, so we have to adjust accordingly

Reply:
To be clear:
Margin between "Commit message" and "Files" became 32px ("Files" margin-top 24px + "Commit message" margin-bottom 8px)
Margin between "Files" and "Metadata" stayed at 24px

I think they should be equal 24px both (or 32px both)

Reply:
you're right, i'll fix that